### ISSUE TITLE
CentOS 6 and Python 2.6

### ISSUE THREAD

**Issue by vdbergh** [niklasf/python-chess#50]
python-chess does not work on python 2.6 because of the Counter class. So it also does not work on CentOS. 

Using this backport 

http://code.activestate.com/recipes/576611-counter-class/

everything seems to work (admittedly I did only limited testing).

It seems a shame that such a useful package as python-chess would not work out of the box on the still widely used CentOS 6 (10 years of support).

Would you consider using the backported Counter class?

**Comment by dsjoerg** [comment]
Might it be easier to install a newer python onto CentOS than to get python-chess to work on an older python?

https://www.digitalocean.com/community/tutorials/how-to-set-up-python-2-7-6-and-3-3-3-on-centos-6-4

**Comment by vdbergh** [comment]
Well getting python-chess to work on Python 2.6 seemed easy enough (although I did not thoroughly test it). The difference between Python 2.6 and Python 2.7 is not that large I think. There are many very large and complicated python packages (e.g. numpy, scipy) that maintain CentOS 6 compatibility.

As far as installing Python 2.7 on CentOS 6. One cannot replace the system python since that is used by ... the system. Of course one can install Python 2.6 and Python 2.7 side by side but that is a bit of a hassle.

**Comment by niklasf** [comment]
Wow, and I thought Debian was a stable distribution :)

It appears that at least also `collections.OrderedDict` is missing. I won't add the backports in python-chess, but I'd be happy to accept patches that fall back to imports from something like https://pypi.python.org/pypi/backport_collections/0.1.

**Comment by vdbergh** [comment]
Ah great. Google didn't tell me this. Much better than adding my own Counter class.

Now as far as doing this seamlessly... It is not clear to me how to do it in a way that would not clutter the source code.  Something like

```
try:
    from collections import Counter
except ImportError:
    from backport_collections import Counter
```

**Comment by niklasf** [comment]
Exactly. Find all relevant places and I'll trust you when you say the test suite is passing on Python 2.6 ;)

**Comment by vdbergh** [comment]
Ok I made a pull request. Please have a look.

**Comment by niklasf** [comment]
Merged #51.
